app/helpers/log.rb

```diff
@@ -0,0 +1,39 @@
+class Log
+
+  def self.info message
+    message = self.format_message message
+    Rails.logger.info message
+  end
+
+  def self.error message
+    message = self.format_message message
+    Rails.logger.error message
+  end
+
+  def self.alert message
+    self.error message
+    self.alert_slack message
+  end
+
+  private
+  def self.format_message message
+    "#{ Time.now.strftime("%Y/%m/%d %H:%M:%S") } #{ message }"
+  end
+
+  def self.alert_slack message
+    uri = URI.parse("https://hooks.slack.com/services/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")
```

Comment:
We need to pull this from an ENV var since it is a secret key. I will also generate a new one since this is public now